css/style.css

```diff
@@ -41,9 +74,9 @@ li {
   line-height: 1.2rem;
 }
 
-.hero a {
+.hero a,
+.tags a {
   color: var(--accent-light, #7f8cff);
-  text-transform: uppercase;
 }
```

Comment:
- [ ] I believe this color would be used for a lot of links in the future, So, make sure to separate it into its own class and add it to the links manually.